pom.xml

```diff
@@ -37,6 +37,12 @@
             <version>${assertj-core.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>RELEASE</version>
```

Comment:
BTW, it's not a best practice to use `RELEASE` version. Usually a specific version is used.